codalab/worker_manager/aws_batch_worker_manager.py

```diff
@@ -7,20 +8,47 @@
 logger = logging.getLogger(__name__)
 
 
+class AWSBatchWorkerManagerConfig:
+    DEFAULT_CONFIG = {
```

Comment:
We don't need this since we already have JSON files in the docker compose files.